dpctl/tensor/_ctors.py

```diff
@@ -202,6 +202,13 @@ def _usm_ndarray_from_suai(obj):
         buffer=membuf,
         strides=sua_iface.get("strides", None),
     )
+    _data_field = sua_iface["data"]
+    if isinstance(_data_field, tuple) and len(_data_field) == 2:
```

Comment:
Should be the comparison relaxed to be prepared for the future possible changes where the length of data field might be increased and some additional information added to the end?
```suggestion
    if isinstance(_data_field, tuple) and len(_data_field) > 1:
```
Since it is only important to protect against out-of-boundary access, it is sufficient that `len(_data_field) > 1`.